doc/radosgw/cloud-restore.rst

```diff
@@ -142,121 +159,133 @@ For example:
             }
         }
     ]
+```
 
 
+Examples of Restore Objects:
+----------------------------
 
-Examples to Restore Object:
----------------------------
-
-Using S3 RestoreObject CLI
-~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Below options of `S3 restore-object <https://docs.aws.amazon.com/cli/latest/reference/s3api/restore-object.html>`__ CLI are supported  -
-
-
-Syntax
-
-::
+Using the S3 RestoreObject CLI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+These options to the `S3 restore-object <https://docs.aws.amazon.com/cli/latest/reference/s3api/restore-object.html>`
+CLI are supported:
```

Comment:
What about "The S3 restore-object CLI supports these options:" ?

Reply:
pushed